src/processTargets/marks/CursorStage.ts

```diff
@@ -4,17 +4,21 @@ import { ProcessedTargetsContext } from "../../typings/Types";
 import { isReversed } from "../../util/selectionUtils";
 import { MarkStage } from "../PipelineStages.types";
 import WeakTarget from "../targets/WeakTarget";
```

Comment:
I tried replacing these lines with `import { TokenTarget, WeakTarget } from "../targets";` but that gives me the following error on extension activation: `Activating extension 'pokey.cursorless' failed: Class extends value undefined is not a constructor or null.`. I tried replacing all references to targets to go through the `target/index` file but no go.

Reply:
Huh. I can take a look in a follow-up